src/Console/Command/Local.php

```diff
@@ -0,0 +1,115 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Console\Command;
+
+use Bref\Bref;
+use Bref\Context\Context;
+use Bref\Runtime\Invoker;
+use Exception;
+use JsonException;
+use Psr\Container\NotFoundExceptionInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Yaml;
+use Throwable;
+
+/**
+ * Local function invocation.
+ */
+class Local
+{
+    public const SIGNATURE = 'local [function] [data] [--file=] [--handler=]';
+
+    public function __invoke(?string $function, ?string $data, ?string $file, ?string $handler, SymfonyStyle $io): int
+    {
+        if ($function && $data && $handler) {
+            throw new Exception('You cannot provide both a funtion name and the --handler= option.');
+        }
+
+        if ($handler) {
+            // Shift the arguments since there is no function passed
+            $data = $function;
+        } else {
+            $handler = $this->handlerFromServerlessYml($function);
+        }
+
+        if ($data && $file) {
+            throw new Exception('You cannot provide both event data and the --file= option.');
+        }
+
+        try {
+            $handler = Bref::getContainer()->get($handler);
+        } catch (NotFoundExceptionInterface $e) {
+            throw new Exception($e->getMessage() . PHP_EOL . 'Reminder: `bref local` can invoke functions that use the FUNCTION runtime, not the HTTP (or "FPM") runtime. If you are unsure, check out https://bref.sh/docs/local-development.html#http-applications to run HTTP applications locally.');
+        }
+
+        if ($file) {
+            if (! file_exists($file)) {
+                throw new Exception("The file '$file' does not exist.");
+            }
+            $data = file_get_contents($file);
+        }
+
+        try {
+            $event = $data ? json_decode($data, true, 512, JSON_THROW_ON_ERROR) : null;
+        } catch (JsonException $e) {
+            throw new Exception('The JSON provided for the event data is invalid JSON.');
+        }
+
+        // Same configuration as the Bref runtime on Lambda
+        ini_set('display_errors', '1');
+        error_reporting(E_ALL);
+
+        $requestId = '8f507cfc-example-4697-b07a-ac58fc914c95';
```

Comment:
It is normal to get the request id here hardcoded ?

Reply:
Yes I wanted to provide one that looks like a real ID, but I thought why generate a random one… I think as a first version that will do well, if anyone actually uses those IDs for something they can always improve that by sending a PR that generates random ones?